### PR TITLE
[TA-403]  feat: update translations script to include all supported locales

### PR DESCRIPTION
**Pull request by SystemDisc**

### CODE REVIEW

File: packages/react-scripts/scripts/translations.js [hunk 1]
```diff
@@ -1,11 +1,41 @@
...
+const manageTranslations = require('react-intl-translations-manager').default;
+const asiagoMessages = path.resolve(paths.asiagoPath, 'dist/messages');
 
 manageTranslations({
   messagesDirectory: asiagoMessages,
   translationsDirectory: 'src/i18n/messages/',
-  languages: ['en', 'en-ca', 'en-uk', 'fr', 'fr-ca', 'de', 'zh'], // any language you need
-})
+  languages: [
+    // any language you need
```
Comment (andela-oakinniranye) [inline packages/react-scripts/scripts/translations.js:11]: @SystemDisc don't we wanna still include the previous translations for backward compatibility?

**Review by morgs32 — APPROVED** [review]
💯

File: packages/react-scripts/scripts/translations.js (hunk 1 above)
Comment (SystemDisc) [reply packages/react-scripts/scripts/translations.js:11]: @andela-oakinniranye I don't think so. This is just for generating a new set of files.

Comment (Jurawa) [reply packages/react-scripts/scripts/translations.js:11]: @andela-oakinniranye we are handling previously supported locales by redirecting them to their current equivalents before loading the app so we dont actually have to include translations for them to keep things simpler.

**Review by Jurawa — APPROVED** [review]

File: packages/react-scripts/scripts/translations.js (hunk 1 above)
Comment (andela-oakinniranye) [reply packages/react-scripts/scripts/translations.js:11]: 👍

**Review by andela-oakinniranye — APPROVED** [review]

**Pull request merged**